sentry_sdk/integrations/featureflags.py

```diff
@@ -0,0 +1,18 @@
+from sentry_sdk.flag_utils import flag_error_processor
+
+import sentry_sdk
+from sentry_sdk.integrations import Integration
+
+
+class FeatureFlagsIntegration(Integration):
```

Comment:
This should probably be called "GenericFeatureFlagIntegration".

Reply:
Should it be consistent with JS? There we have `featureFlagsIntegration`

Reply:
It should be consistent between both but "FeatureFlagsIntegration" is too vague.  A user may think they need to install this to use LD or OF.

Reply:
"Custom" or "Generic" communicates what the integration is for.

Reply:
I don't think it's needed - we rarely have integrations that depend on each other (replay and replay canvas being the one exception I know of). Also, users are likely to stumble upon these integrations from docs where they'll be copy and pasting anyway.  

That being said, I care less about the specific name, and more about it being consistent across SDKs

Reply:
Decided on slack to go with just FeatureFlags